AI/app/rag/attraction_search.py

```
@@ -0,0 +1,105 @@
+        if not line:
+            continue
+        if ':' in line:
+            key, value = line.split(':', 1)
+            key = key.strip()
+            value = value.strip()
+            # 숫자 변환 시도
+            if key in ["attraction_id", "category_id"]:
+                try:
+                    value = int(value)
+                except:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)